lainnya/abstrak-en.tex

```diff
@@ -0,0 +1,37 @@
+\begin{center}
+	\large
+  \textbf{\emph{ANTI-GRAVITY} BASED ENERGY CALCULATION ON OUTER SPACE ROCKETS}
+\end{center}
+% Menyembunyikan nomor halaman
+\thispagestyle{empty}
+
+\begin{flushleft}
+    \setlength{\tabcolsep}{0pt}
+    \bfseries
+    \begin{tabular}{lc@{\hspace{6pt}}l}
+    Student Name / NRP&: &Elon Reeve Musk / 0123204000001\\
+    Department&: &Aerospace Engineering FTD - ITS\\
+    Advisor&: &1. Nikola Tesla, S.T., M.T.\\
+    & & 2. Wernher von Braun, S.T., M.T.\\
+    \end{tabular}
+    \vspace{4ex}
+\end{flushleft}
+\textbf{Abstract}
+
+% Isi Abstrak
+Suspension is an important component in vehicles because it plays an important role in
+maintaining comfort and safety while driving. A new idea was introduced, namely, Series
+Active Variable Geometry Suspension (SAVGS), where this suspension system has better
+performance than passive suspension and can overcome the weaknesses of active suspension.
+Research continues to improve the performance of SAVGS. The effect of linkage length (single
+link) on SAVGS performance, especially comfort and stability, is studied. A quarter car is used
+to model the dynamics of the vehicle suspension system. The effect of single link length is
+analyzed in the form of stiffness coefficient and damping coefficient. The linear model is used
+to design the state-feedback control system (LQR). The performance of the control system was
+tested on a nonlinear model made using Simscape Multibody. The simulation results show that
+the longer the single link used, the greater the vehicle's comfort and stability. However, the
+longer the single link required more considerable control input.
```

Comment:
please use `\lipsum` for the rest of the abstract content

reason: since it's just a template, let's make the source code as small as possible.